convert2rhel/utils.py

```diff
@@ -620,4 +627,52 @@ def set_locale():
     os.environ.update({"LC_ALL": "C", "LANG": "C"})
 
 
+def options_from_config_files(config_opts, cfg_path=None):
+    """Get the options from config file.
+
+    The order is:
+    1. if filename is set then load from it
+    2. ~/.convert2rhel.ini
+    3. /etc/convert2rhel.ini
+
+    Return config_opts filled with loaded values.
+    """
+    headers = ["subscription_manager"]  # supported sections in config file
+    config_file = configparser.ConfigParser()
+    paths = CONFIG_PATHS
+    # create dict with all supproted options, all of them set to None
+    # needed for avoiding problems with files priority
+    supproted_opts = {"password": None, "activation_key": None}
+
+    if cfg_path:
+        if not os.path.exists(cfg_path):
+            raise FileNotFoundError
```

Comment:
Good thinking that this particular case (the cfg_path passed from the user) should raise an exception whereas the other config file location should not!  A couple changes to make

* The exception should have a message.
* FileNotFoundError doesn't exist in python2.x.  You could just use OSError here (which is what FileNotFoundError was in Python-2.  But if you do that you'll want to make sure that you populate all the fields that OSError has.  Maybe the easiest way to see what it should look like is to make an OSError onpurpose and then fill in your raise appropriately (Catch the exception from `open('/nonexistant')` and then use that as a model for what you need to raise.)